gen/template/Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+gem 'cli-kit', '~> 3.0.0.beta'
```

Comment:
Can we use template variables to keep these up to date as `CLI::Kit::VERSION` and `CLI::UI::VERSION?

Reply:
How would you pull these variables in if the gems dont exist?

Reply:
The generator _is_ a `CLI::Kit` app itself, so it has these.

Reply:
Yeah, I thought about this and didn't bother, but it is probably a good idea. will revise.

Reply:
right, since this is a template. This will have to be a template variable we inject then.